api/src/main/resources/marquez/db/migration/V41__alter_jobs_add_job_parent_id.sql

```diff
@@ -0,0 +1,33 @@
+ALTER TABLE jobs ADD COLUMN IF NOT EXISTS parent_job_id uuid CONSTRAINT jobs_parent_fk_jobs REFERENCES jobs (uuid);
+ALTER TABLE runs ADD COLUMN IF NOT EXISTS parent_run_id uuid CONSTRAINT runs_parent_fk_runs REFERENCES runs (uuid);
+DROP INDEX IF EXISTS jobs_name_index;
```

Comment:
How expensive would it be to keep the index? (Wondering if that's just a good index to have).

Reply:
It's replaced with a different index